api/hooks/initPassport.js

```diff
@@ -69,10 +70,12 @@ module.exports = function (sails) {
                      if (err) {
                         console.error("Failed to authenticate user", err);
```

Comment:
When trying to debug specific route attempts, it is always helpful to be able trace which request this error is related to.

This is where the `req.log()` comes in.  It will print the `jobID` for the current request for each log entry.

So try to use `req.log()` instead of `console.*()` instead.